pinot-core/src/main/java/org/apache/pinot/core/query/request/context/QueryContext.java

```diff
@@ -74,6 +74,7 @@
   private final List<ExpressionContext> _selectExpressions;
   private final List<String> _aliasList;
   private final FilterContext _filter;
+  private final ExpressionContext _filterExpression;
```

Comment:
This field is not used, let's remove it. Filter shouldn't be stored as `ExpressionContext`